File: PYTHON-PRAKHAR VERMA/prime.py
num=int(input("ENTER THE NUMBER: "))
if num>1:
    for i in range(2,num):
        if(num%i)==0:
            print(num,"IS NOT A PRIME NUMBER")
            break
    else:
        print(num,"IS A PRIME NUMBER")


        # b=num%10
        # rev=rev*10+b
        # num=num//10

File: COMPUTER PROGRAMMING IN C AND C++/prime.c
#include <stdio.h>
int main()
{
	int n,i,count;
	printf("ENTER A NUMBER TO BE CHECKED:");
	scanf("%d",&n);
	for(i=2;i<n;i++)
	{
		if(n%i==0)
		{
			count=1;
			break;
		}
	}
	if (count==1)
	{
		printf("PRIME");
	}else {
		printf("NOT PRIME");
	}
	return 0;
}
